scripts/helm/install

```diff
@@ -31,8 +31,8 @@ serviceName="ffc-demo-user-service"
     --atomic \
     --install \
     --namespace ${kubernetesNamespace} \
-    --values helm/development-values.yaml \
+    --values helm/ffc-demo-user-service/development-values.yaml \
     --wait \
-    ${serviceName} \
-    ./helm
+    ffc-demo-user-service \
```

Comment:
this should probably stay as `$(serviceName}`

Reply:
I've changed all the hard coded paths, to use the ${serviceName} parameter instead. Not sure this is the best approach, but we're using the servicename in the path.